infra/recipes/k8s/helm/tables/config/azure/sandbox/cluster.yaml

```diff
@@ -0,0 +1,26 @@
+cluster:
```

Comment:
Is there any difference between tables and housetables cluster.yaml. If not, we can create a symlink for housetables cluster.yaml?

Reply:
The differences are with `previous-versions-max` and `security.token.interceptor`: [tables cluster](https://github.com/kmcclenn/openhouse/blob/services_deployed_helm/infra/recipes/k8s/helm/tables/config/azure/sandbox/cluster.yaml#L12C1-L23C40) versus [hts cluster](https://github.com/kmcclenn/openhouse/blob/services_deployed_helm/infra/recipes/k8s/helm/housetables/config/azure/sandbox/cluster.yaml#L12,L23). I am not sure if they are significant though or what they really mean. What do they do/change about the cluster?

Reply:
I think cluster.yaml should be same for both the services. (cc: @HotSushi )

Reply:
Yes we don't have `com.linkedin.openhouse.common.security.DummyTokenInterceptor` for housetables.

Reply:
Should be good to keep two separate configs.

Reply:
That is correct today we have same cluster yaml for both tables and housetables. Even jobs iirc.

Reply:
I would prefer/recommend keeping the same cluster.yaml file. That way each service uses what it needs and we don't evolve into a service-specific cluster.yaml

Reply:
Okay I will symlink